collector/src/bin/collector.rs

```diff
@@ -1187,6 +1201,105 @@ Make sure to modify `{dir}/perf-config.json` if the category/artifact don't matc
             println!("Data of artifact {name} were removed");
             Ok(0)
         }
+        Commands::BenchLocalDiff { db, a_id, b_id } => {
+            let pool = Pool::open(&db.db);
```

Comment:
Please move this already to e.g. `src/compare.rs`. This logic will probably grow much bigger than this in time :) We can keep the DB connection here, and send the DB connection and the two artifact names to a function that will do the comparison.